backend/command/src/shared/functions/get-stringified.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { getStringified } from './get-stringified';
+
+describe('getStringified', () => {
+  it('should return stringified object with multiple lines', () => {
+    const obj = {
+      query: 'query',
+      tags: ['tag1', 'tag2'],
+    };
+    expect(getStringified(obj)).toEqual(
```

Comment:
Is it just me or does this need formating?

Reply:
Nah thats unfortunately the case that if you make a multiline string it counts the tabs from the beginning, so basically the first `{` is formatted by ESLint, and the rest is the correct distance from the left side xD